### PR TITLE
Fix balance in example code

### LINKED ISSUE #6018: restoration.wiener doc example gives poor result
## Description
I had a look at the [Example code](https://scikit-image.org/docs/dev/api/skimage.restoration.html#skimage.restoration.wiener) for Wiener-Hunt deconvolution and the result I get looks blurrier than the image it starts with. Is this the expected behavior? I played around with the balance parameter but it didn't help. I am trying to understand if this is the deconvolution that would be applicable to my problem.

wiener

## Way to reproduce
```python
from skimage import color, data, restoration
img = color.rgb2gray(data.astronaut())
from scipy.signal import convolve2d
psf = np.ones((5, 5)) / 25
conv = convolve2d(img, psf, 'same')
rng = np.random.default_rng()
noised = conv + 0.1 * conv.std() * rng.standard_normal(conv.shape)
deconvolved_img = restoration.wiener(noised, psf, 1100)


fig, ax = plt.subplots(nrows=1, ncols=4, figsize=(8, 6))
plt.gray()


ax[0].imshow(img)
ax[0].set_title('Original')
ax[0].axis('off')

ax[1].imshow(conv)
ax[1].set_title('Convoluted')
ax[1].axis('off')

ax[2].imshow(noised)
ax[2].set_title('Noised')
ax[2].axis('off')

ax[3].imshow(deconvolved_img)
ax[3].set_title('Restoration using Wiener')
ax[3].axis('off')

fig.subplots_adjust(wspace=0.02, hspace=0.2,
                    top=0.9, bottom=0.05, left=0, right=1)
plt.show()
```

## Version information

```python
3.9.7 (default, Sep 16 2021, 13:09:58) 
[GCC 7.5.0]
Linux-3.10.0-1160.36.2.el7.x86_64-x86_64-with-glibc2.17
scikit-image version: 0.18.1
numpy version: 1.21.2

```

### PR DESCRIPTION
**Pull request by elena-pascal**
## Description

Closes #6018.

Fix for small inconsistencies in documentation example as discussed #6018

## For reviewers

- Check that the PR title is short, concise, and will make sense 1 year
  later.
- Check that new functions are imported in corresponding `__init__.py`.
- Check that new features, API changes, and deprecations are mentioned in
  `doc/release/release_dev.rst`.
- There is a bot to help automate backporting a PR to an older branch. For
  example, to backport to v0.19.x after merging, add the following in a PR
  comment: `@meeseeksdev backport to v0.19.x`
- To run benchmarks on a PR, add the `run-benchmark` label. To rerun, the label
  can be removed and then added again. The benchmark output can be checked in
  the "Actions" tab.

### CODE REVIEW

File: doc/examples/filters/plot_restoration.py [hunk 1]
```diff
@@ -4,29 +4,33 @@
...
 the noise power and the image frequency power.
 
 .. [1] François Orieux, Jean-François Giovannelli, and Thomas
        Rodet, "Bayesian estimation of regularization and point
        spread function parameters for Wiener-Hunt deconvolution",
        J. Opt. Soc. Am. A 27, 1593-1607 (2010)
+
+       https://www.osapublishing.org/josaa/abstract.cfm?URI=josaa-27-7-1593
+
+       https://hal.archives-ouvertes.fr/hal-00674508/document
```
Comment (mkcor) [inline doc/examples/filters/plot_restoration.py:33]:
```suggestion
       https://www.osapublishing.org/josaa/abstract.cfm?URI=josaa-27-7-1593
       https://hal.archives-ouvertes.fr/hal-00674508
```

File: skimage/restoration/deconvolution.py [hunk 2]
```diff
@@ -105,7 +105,7 @@ def wiener(image, psf, balance, reg=None, is_real=True, clip=True):
 
            https://www.osapublishing.org/josaa/abstract.cfm?URI=josaa-27-7-1593
 
-           http://research.orieux.fr/files/papers/OGR-JOSA10.pdf
+           https://hal.archives-ouvertes.fr/hal-00674508/document
```
Comment (mkcor) [inline skimage/restoration/deconvolution.py:108]:
```suggestion
           https://hal.archives-ouvertes.fr/hal-00674508
```
(Taking the reader only to the landing page, where they can easily download the PDF file...)

File: skimage/restoration/deconvolution.py [hunk 3]
```diff
@@ -241,7 +241,7 @@ def unsupervised_wiener(image, psf, reg=None, user_params=None, is_real=True,
...
```
Comment (mkcor) [inline skimage/restoration/deconvolution.py:244]:
```suggestion
           https://hal.archives-ouvertes.fr/hal-00674508
```

**Review by mkcor — APPROVED** [review]
Thank you so much! I really appreciate and highly value these fixes, which improve the quality of our documentation.

**Comment by elena-pascal** [comment]
Thank you for encouraging words and walking em through things. I learned a lot!

**Comment by mkcor** [comment]
Thanks for being so prompt, @elena-pascal!

PS: You can also apply suggested changes as a *batch*: https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request

**Comment by mkcor** [comment]
Closes #6018

**Review by rfezzani — APPROVED** [review]
Thank you @elena-pascal! Merging :tada:

**Pull request merged**